change/@ni-nimble-components-37c84290-c5e8-4c31-a22e-e69547cc96c0.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "default buttons appearance updates (#663)",
+  "packageName": "@ni/nimble-components",
+  "email": "103057880+aidendk@users.noreply.github.com",
+  "dependentChangeType": "patch"
+}
```

Comment:
I'm a bit wary of the Chromatic status check. Do we know why the UI Tests are showing as unchanged? Is that because you already approved the diffs in a previous build, so the latest build is unchanged relative to that? Or is Chromatic somehow confused (I did change how Chromatic runs in #1301 so I want to make sure I didn't break something).

If you have an explanation, that's great. If not then it wouldn't hurt to do a sanity check in another clean branch where you 
1. make a change that triggers a diff in a build
2. accept that diff
3. make another change
4. see whether the diff is reported on that build anymore

![image](https://github.com/ni/nimble/assets/10500124/d098acda-eb26-4f3d-ab3a-4e81e6275080)